sources/lib/Model/ContainerModelLayerPooler.php

```diff
@@ -11,8 +11,8 @@
 
```

Comment:
I don’t undestand the goal of changes in this file. The `ContainerAwareTrait` already requires a `ContainerInterface`.